runtime/src/accounts_index.rs

```diff
@@ -23,6 +30,26 @@ pub type RefCount = u64;
 pub type AccountMap<K, V> = BTreeMap<K, V>;
 
 type AccountMapEntry<T> = Arc<AccountMapEntryInner<T>>;
+type SecondaryIndexEntry = DashMap<Pubkey, RwLock<HashSet<Slot>>>;
+
+enum ScanTypes<R: RangeBounds<Pubkey>> {
+    Standard(Option<R>),
+    Indexed(IndexKey),
+}
+
+#[derive(Debug, Clone, Copy)]
+pub enum IndexKey {
+    ProgramId(Pubkey),
+    TokenOwner(Pubkey),
+    Mint(Pubkey),
```

Comment:
Make these names SPL token specific to avoid future confusion:
```suggestion
    SplTokenOwner(Pubkey),
    SplTokenMint(Pubkey),
```

Reply:
Done in https://github.com/solana-labs/solana/pull/14212/commits/d204a08432c3936cd1450cf16e1a67236e758461